packages/core/src/browser/keys.ts

```diff
@@ -302,26 +302,24 @@ export class KeyCode {
         }
     }
 
-    public static toCode(event: KeyboardEvent): string {
+    public static toCode(event: KeyboardEvent & { readonly keyIdentifier?: string }): string {
```

Comment:
I wondered where `keyIdentifier` came from.  Perhaps add a comment like:

```
// keyIdentifier is used to access this deprecated field:
// https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyIdentifier
```

Reply:
Yes, you're right. But it was there before as well; we just made an explicit cast to `any`.

Reply:
Ahh, I see.